packages/react/src/provider.tsx

```diff
@@ -70,8 +70,10 @@ export const LogtoProvider = ({ logtoConfig, children }: LogtoProviderProperties
 
       dispatch({ type: 'HANDLE_CALLBACK_REQUEST' });
       try {
+        const { redirectUri = '' } = logtoClient.sessionManager?.get() ?? {};
         await logtoClient.handleCallback(uri);
         dispatch({ type: 'HANDLE_CALLBACK_SUCCESS', payload: { claims: logtoClient.getClaims() } });
+        history.replaceState(history.state, '', redirectUri);
```

Comment:
nit: if `redirectUri` is empty, stop replaceState, although it won't happen theoretically.

Reply:
make sense